### PR TITLE
Add i18n & withState React helpers

### PR DESCRIPTION
**Pull request by goldoraf**
...and some very little styling additions :guitar:

### CODE REVIEW

File: react/helpers/withState.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/** @jsx h */
+import { h, Component } from 'preact'
```
Comment (y-lohse) [inline react/helpers/withState.jsx]: Why is it importing from `preact` and not `react`here?

Comment (goldoraf) [inline react/helpers/withState.jsx]: Just an artefact from a not so distant past :), fixed!

**Comment by goldoraf** [comment]
I use `withState` in cozy-settings and my-accounts, we'll probably find other uses of it.

File: stylus/ui-app/forms.styl [hunk 2]
```diff
@@ -11,9 +11,11 @@ checkbox-size = 20px
 
 $form
     .coz-form
-        position relative
         margin   1em 0 .5em
 
+    .coz-form
+    .coz-form-group
```
Comment (GoOz) [inline stylus/ui-app/forms.styl]: What is the use case for a new class? I can't really see the need without an example :(

Comment (goldoraf) [inline stylus/ui-app/forms.styl]: Well, the use of flexbox on the form elements force us to stack labels and inputs, but sometimes an "input" may be composed of many elements, and I would like to group these elements into a div that flexes the same way than `coz-form`. Does it make sense to you?

Comment (GoOz) [inline stylus/ui-app/forms.styl]: It does 😄 
I'll wait to see it in action to make my mind but so far that seems like a good reason to add a new class indeed. 😉

**Pull request merged**